pages/bookmarks/index.js

```diff
@@ -1,12 +1,10 @@
 import styled from "styled-components";
 
-export default function BookmarksPage() {
+export default function BookmarksPage(/* {isBookmarked} */) {
```

Comment:
one tiny peace of commented code left over, please remove ;)

Reply:
resolved